SofascoreAcademy/SofascoreAcademy/View/TabItemView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  TabItemView.swift
+//  SofascoreAcademy
+//
+//  Created by Anton Pomper on 26.03.2024..
+//
+
+import UIKit
+import SnapKit
+import SofaAcademic
+
+class TabItemView: BaseView {
+    
+    private let imageView = UIImageView()
+    private let titleLabel = UILabel()
+    
+    override func addViews() {
+        super.addViews()
+        addSubview(imageView)
+        addSubview(titleLabel)
+    }
+    
+    override func styleViews() {
+        super.styleViews()
+        
+        backgroundColor = .colorPrimaryDefault
+        
+        imageView.contentMode = .scaleAspectFit
+        imageView.clipsToBounds = true
+        
+        titleLabel.textAlignment = .center
+        titleLabel.font = titleLabel.font.withSize(14)
+        titleLabel.textColor = .surfaceSurface1
+        titleLabel.numberOfLines = 1
+    }
+    
+    override func setupConstraints() {
+        super.setupConstraints()
+        
+        let screenWidth = UIScreen.main.bounds.width
```

Comment:
ovo nije dobro, vrlo su rijetke situacije gdje se koristi UIScreen, a i kad se koristi, to je u nekom VC-u, sigurno ne u view.
Ovaj view mora samo definirati svoje elemente, a njegovu širinu kontrolira njegov parent. Tako da maknite width constrainte odavde. Ovaj view se stavlja u stack view koji automatski pazi na širinu svojih childova. Ako dobro konfigurirate stack view, čak će i rezervirati jednak prostor za svoje childove pa istražite koji je to property ;)